cassandra/pom.xml

```diff
@@ -27,13 +27,14 @@
     </parent>
 
     <groupId>org.apache.zeppelin</groupId>
-    <artifactId>zeppelin-cassandra_2.10</artifactId>
+    <artifactId>zeppelin-cassandra_${scala.binary.version}</artifactId>
```

Comment:
here for instance, should artifactId match the actual version of scala used, ie. `cassandra.scala.version`?